### PR TITLE
Add Matt Stephenson PulumiUP blog post

### PR DESCRIPTION
**Pull request by gthuang**
## Description
A blog post summarizing Matt Stephenson's PulumiUP presentation.

**To do:**

- [x] need to add the correct YouTube video once it is published by @SaraDPH 

## Checklist:

- [x] I have reviewed the [style guide](https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md).
- [x] If blogging, I have reviewed the [blogging guide](https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md).
- [ ] I have manually confirmed that all new links work.
- [ ] I added aliases (i.e., redirects) for all filename changes.
- [ ] If making css changes, I rebuilt the bundle.

### CODE REVIEW

**Review by scottslowe — APPROVED** [review]
Approved (pending the addition of the correct YouTube video, of course!)

**Comment by SaraDPH** [comment]
@gthuang the link is https://youtu.be/t-oSFZuNqXQ?rel=0

File: themes/default/content/blog/how-starburst-data-creates-infrastructure-automation-magic-with-code/index.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+* **Support for programming languages:** Pulumi supports the main languages used by Starburst’s developers, Java and TypeScript. This enables all engineers to write infrastructure code and promotes a cohesive application and infrastructure codebase.
+* **Pulumi Automation API:** Automation API is a programmatic interface for the Pulumi CLI and Pulumi IaC engine, which enables Starburst engineers to build [infrastructure automation](https://github.com/pulumi/automation-api-examples) into their applications and CI/CD processes, and orchestrate complex provisioning workflows.
+
+## Benefits of Using Pulumi
+Adopting Pulumi and its Automation API provided several benefits that made Starburst’s engineers more efficient and productive, ultimately leading to faster and more reliable releases for Starburst’s customers. Matt summed up the benefits of Pulumi for Starburst’s engineers: "_When I think about building an airplane, I think about precision and having the right tools for the job._" Pulumi, for Starburst, epitomized this precision and efficiency.
```
Comment (cnunciato) [inline themes/default/content/blog/how-starburst-data-creates-infrastructure-automation-magic-with-code/index.md:63]: Just the quotes without the italics is enough (it's generally one or the other):

```suggestion
Adopting Pulumi and its Automation API provided several benefits that made Starburst’s engineers more efficient and productive, ultimately leading to faster and more reliable releases for Starburst’s customers. Matt summed up the benefits of Pulumi for Starburst’s engineers: "When I think about building an airplane, I think about precision and having the right tools for the job." Pulumi, for Starburst, epitomized this precision and efficiency.
```

File: themes/default/content/blog/how-starburst-data-creates-infrastructure-automation-magic-with-code/index.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+**Consistent deployment lifecycles:** Using Pulumi’s [integration with GitHub Actions](/docs/using-pulumi/continuous-delivery/github-actions/), Starburst engineers built a streamlined CI/CD process where infrastructure and application code builds within the same pipelines with automated test environments. They accomplished this using Pulumi Automation API to abstract the underlying runtime environments, circumventing the need for additional layers and wrappers.
+
+## Try Pulumi for Infrastructure as Code Automation
+
+[Sign up for a free account](https://app.pulumi.com/signup) to try deploying infrastructure on any cloud with [Pulumi Deployments](/docs/pulumi-cloud/deployments/), or [register for an upcoming workshop](https://www.pulumi.com/resources/#upcoming) to learn more about how Pulumi can help you ship cloud infrastructure faster and more safely.
+
+[Go to more case studies](/case-studies/)
```
Comment (cnunciato) [inline themes/default/content/blog/how-starburst-data-creates-infrastructure-automation-magic-with-code/index.md:75]:
```suggestion
[Go to more case studies &rarr;](/case-studies/)
```

**Pull request merged**